ros/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "lib/inertial-sense-sdk"]
```

Comment:
This should be removed, since ros does not have submodules.

Reply:
Done.  Thanks.